dmsite/venv/src/ProfileComponent.js

```diff
@@ -1,29 +1,52 @@
 import React, { Component } from 'react';
 import { withRouter } from 'react-router-dom';
-import { Container, Row, Col } from 'reactstrap';
 import FilterColumn from './DataColumn/FilterColumn';
 import DisplayData from './DataColumn/DisplayData';
 import DisplayColumn from './VisualColumn/DisplayColumn';
 import Results from './ClassificationColumn/Results';
+import { Container, Card, Button,CardTitle, CardBody, InputGroup, InputGroupText, InputGroupAddon, Input} from 'reactstrap';
 
 class Profile extends Component{
     render(){
         return (
-            <Container fluid>
-                <h1>Campaign: Current Campaign</h1>
-                <p> This campaign organizes Data1, Data2, Data3 </p>
-                <Row>
-                    <Col md="3">
-                        <FilterColumn name={'Datasets'} />
-                        <DisplayData name={'File'}/>
-                    </Col>
-                    <Col md="5">
-                        <Results name={"Classifications"} />
-                    </Col>
-                    <Col md="4">
-                        <DisplayColumn name={'Display Actions'} />
-                    </Col>
-                </Row>
+            <Container fluid style={{ justifyContent: 'center' }}>
+              <Card style={{ width: '60%', justifyContent: 'center', alignSelf: "center" }}>
```

Comment:
It peeves me that you use single quotes for the first 2 values and double quotes in the alignSelf. Try to keep things consistent.